marconi-chain-index/src/Marconi/ChainIndex/Indexers/Utxo.hs

```diff
@@ -742,6 +753,16 @@ instance Buffered UtxoHandle where
                    )
                    VALUES (?, ?)|]
             datumRows
+          `concurrently_` do
+            SQL.execute
```

Comment:
Do we need a delete statement to remove old chain tips? We only need the most recent one.

Reply:
I was sure it was added, adding it again, for real this time.

Reply:
I decided to use a constraint to ensure that there's only one row and used `INSERT OR REPLACE`, it's safer and cleaner than `DELETE/INSERT`.

Reply:
That works